k8-util/docker/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+
+if [ -n "$MINIKUBE_DOCKER_ENV" ]; then
+  eval $(minikube -p minikube docker-env)
+fi
+
+tmp_dir=$(mktemp -d -t fluvio-docker-image-XXXXXX)
+cp target/x86_64-unknown-linux-musl/$CARGO_PROFILE/fluvio $tmp_dir/
```

Comment:
can you check if this script works in Mac?,

Reply:
I don't have access to one. Can you pull locally and run smoke-test-k8?

Reply:
works